index.html

```diff
@@ -18,24 +19,68 @@
 </head>
 <body>
 
-<div class="container-fluid">
-    <h1>EUMIS Sumission management</h1>
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container">
+        <div class="col-sm-6">
+            <img src="images/logo.jpg" id="se-logo">
+        </div>
+        <div class="col-sm-6">
+            <a class="navbar-brand" href="#">EUMIS Submissions</a>
+        </div>
+      </div>
+      <!--<div id="navbar" class="collapse navbar-collapse">
+        <ul class="nav navbar-nav">
+          <li><a href="#">Home</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>--><!--/.nav-collapse -->
+    </div>
+
+  </nav>
+
+<!--<div id="background-strip"></div>-->
+<div class="container" style="padding-top:2em;">
+
 <div class="row">
         <div class="col-sm-6">
-            <h2>Submissions</h2>
+            <h2>EUMIS Submission management</h2>
+            <h2><small>Submit financial data about your projects online</small></h2>
```

Comment:
Thanks, amended to <h2>EUMIS Submission management</br><small>Submit financial data about your projects online</small></h2>